Description: Remove bundler usage
 Remove the usage of bundler for finding out gems
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,5 +1,3 @@
-Bundler.require
-
 require 'get_process_mem'
 require 'test/unit'
 
@@ -7,4 +5,4 @@
   path = Pathname.new(File.expand_path("../fixtures", __FILE__))
   return path.join(name) if name
   path
-end
\ No newline at end of file
+end
